rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/docker/DockerBakeHandlerSpec.groovy

```diff
@@ -150,6 +150,7 @@ class DockerBakeHandlerSpec extends Specification implements TestDefaults {
                                         package_name: PACKAGES_NAME,
                                         base_os: "ubuntu",
                                         request_id: SOME_UUID,
+                                        commit_hash: SOME_COMMIT_HASH,
```

Comment:
Might be good to add a test case demonstrating that `docker_target_image_tag` is always set to _something_. If there's no `commit_hash`, and no explicit `docker_target_image_name`, just verify that the tag was assigned some non-null value? Don't see any need to mock the clock or anything like that.